tools/setupResourceScripts/files/OMOPDataset/concept_ancestor.json

```diff
@@ -1,5 +1,12 @@
 {"ancestor_concept_id": 49, "descendant_concept_id": 0, "min_levels_of_separation": 1, "max_levels_of_separation": 1}
 {"ancestor_concept_id": 49, "descendant_concept_id": 4326177, "min_levels_of_separation": 1, "max_levels_of_separation": 1}
+{"ancestor_concept_id": 4326177, "descendant_concept_id": 4187458, "min_levels_of_separation": 1, "max_levels_of_separation": 1}
+{"ancestor_concept_id": 4326177, "descendant_concept_id": 4163872, "min_levels_of_separation": 1, "max_levels_of_separation": 1}
+{"ancestor_concept_id": 4163872, "descendant_concept_id": 4196006, "min_levels_of_separation": 1, "max_levels_of_separation": 1}
+{"ancestor_concept_id": 4187458, "descendant_concept_id": 4187458, "min_levels_of_separation": 0, "max_levels_of_separation": 0}
+{"ancestor_concept_id": 4326177, "descendant_concept_id": 4326177, "min_levels_of_separation": 0, "max_levels_of_separation": 0}
+{"ancestor_concept_id": 4163872, "descendant_concept_id": 4163872, "min_levels_of_separation": 0, "max_levels_of_separation": 0}
+{"ancestor_concept_id": 4196006, "descendant_concept_id": 4196006, "min_levels_of_separation": 0, "max_levels_of_separation": 0}
```

Comment:
I think it's fine to commit this, but just as a heads up, if we re-run the populate script, it will get overwritten.

Reply:
I think it makes sense to either:
1. Not commit this
2. Commit this and file a ticket to update the populate script
3. Update the populate script to include some more information here.

If you have not yet filed a ticket to update the populate script, would you mind doing that? I would just hate for someone to run the populate script and then be surprised when they have a larger diff than they were expecting.

Reply:
I agree the script should be updated! I just filed a ticket to update the script!

Reply:
https://broadworkbench.atlassian.net/browse/DC-969